src/components/Elements.tsx

```diff
@@ -112,38 +112,23 @@ export const Elements: FunctionComponent<PropsWithChildren<ElementsProps>> = (({
   options,
   children,
 }: PrivateElementsProps) => {
-  const final = React.useRef(false);
-  const isMounted = React.useRef(true);
   const parsed = React.useMemo(() => parseStripeProp(rawStripeProp), [
     rawStripeProp,
   ]);
+
```

Comment:
This file has the biggest set of changes, but behaviorally it's the same. Rather than conditionally triggering side effects in render, they're now in useEffects.